packages/components/toggle-group/src/toggle-group.stories.ts

```diff
@@ -361,3 +383,213 @@ export const All: Story = {
     </div>
   `
 };
+
+export const All: Story = {
+  render: () => {
+    const renderRow = (options: {
+      fill: ToggleGroupFill;
+      emphasis: ToggleGroupEmphasis;
+      content?: 'button' | 'text';
+    }) => {
+      const buttons = (buttonoptions: string[]) => {
+        return html`<sl-toggle-button aria-label="Bold" ?pressed=${buttonoptions.includes('pressed')}>
+            <sl-icon name="far-bold" slot="default"></sl-icon>
+            <sl-icon name="fas-bold" slot="pressed"></sl-icon>
+          </sl-toggle-button>
+          <sl-toggle-button aria-label="Italic" ?pressed=${buttonoptions.includes('pressed')}>
+            <sl-icon name="far-italic" slot="default"></sl-icon>
+            <sl-icon name="fas-italic" slot="pressed"></sl-icon>
+          </sl-toggle-button>
+          <sl-toggle-button aria-label="Underline">
+            <sl-icon name="far-underline" slot="default"></sl-icon>
+            <sl-icon name="fas-underline" slot="pressed"></sl-icon>
+          </sl-toggle-button>`;
+      };
+      const text = (buttonoptions: string[]) => {
+        return html`<sl-toggle-button aria-label="Bold" ?pressed=${buttonoptions.includes('pressed')}>
+            Read
+          </sl-toggle-button>
+          <sl-toggle-button aria-label="Italic"> Write </sl-toggle-button>`;
+      };
+      return html`
+        <tr>
+          <th>${options.emphasis} - ${options.fill}</th>
+          <td>
+            <sl-toggle-group multiple fill=${ifDefined(options.fill)} emphasis=${ifDefined(options.emphasis)} size="sm">
+              ${options.content === 'text' ? text(['pressed']) : buttons(['pressed'])}
+            </sl-toggle-group>
+            <sl-toggle-group
+              multiple
+              fill=${ifDefined(options.fill)}
+              emphasis=${ifDefined(options.emphasis)}
+              shape="pill"
+              size="sm"
+            >
+              ${options.content === 'text' ? text(['pressed']) : buttons(['pressed'])}
+            </sl-toggle-group>
+          </td>
+          <td>
+            <sl-toggle-group
+              multiple
+              fill=${ifDefined(options.fill)}
+              emphasis=${ifDefined(options.emphasis)}
+              disabled
+              size="sm"
```

Comment:
Is it intentional that in `sm` size the width changes when I press the button?

<img width="339" alt="image" src="https://github.com/user-attachments/assets/62ca126c-e6f5-40a9-bfa9-8860cb377593" />

<img width="367" alt="image" src="https://github.com/user-attachments/assets/39389bcc-4f7f-4e3c-9a6a-f48fcb7614dc" />